src/test/java/com/uber/cadence/workflow/WorkflowTest.java

```diff
@@ -3216,12 +3006,7 @@ public WorkflowStub newUntypedWorkflowStub(
                 })
             .setDomain(DOMAIN)
             .build();
-    WorkflowClient wc;
-    if (useExternalService) {
-      wc = WorkflowClient.newInstance(service, clientOptions);
-    } else {
-      wc = testEnvironment.newWorkflowClient(clientOptions);
-    }
+    WorkflowClient wc = cadenceTestRule.createWorkflowClient(clientOptions);
```

Comment:
nit: maybe add client options to builder?

Reply:
Added, good call. This test still needs to create a custom client to test out its interceptor but that's a very useful thing to expose.